add `importFiles` api (#50)
webxdc/webxdc_docs: src/spec.md

> If you want to store text don't use `btoa()`, rather use `message.file.plainText` directly, because [`btoa()` has problems with some unicode/emojis](https://developer.mozilla.org/en-US/docs/Web/API/btoa#unicode_strings)

### importFiles()

```js

> let files = await window.webxdc.importFiles(filters);

Reviewer: ```suggestion
let files = await window.webxdc.importFiles(options);
```

i think, "filters" is not really matching here, also in its plural form (it is maybe inspired by https://www.electronjs.org/docs/latest/api/dialog - but there there user can really select one filter from several defined ones - but this is not what we want here)

another name would be "accept". otoh, it is not super-important as it is not really part of the api the user "types" at the end, so no blocker

```

> `sendToChat()` (requires Delta Chat 1.38 or compatible) allows an webxdc app to import files.

Reviewer 2: ```suggestion
`sendToChat()` (requires Delta Chat 1.38 or compatible) allows a webxdc app to import files.
```

> Dependenfing on platform support this just opens the system filepicker or a custom one.

Reviewer: ```suggestion
Depending on platform support this just opens the system filepicker or a custom one.
```

> Dependenfing on platform support this just opens the system filepicker or a custom one.

Reviewer 2: ```suggestion
Depending on platform support this just opens the system file chooser or a custom one.
```

This custom file picker should show recent attachments that were received and sent, to make importing of a file that you just received from someone easier (you don't need to save it to the filesystem first), but it also still shows a button to open the system file picker.

- `filters`: an object with the following properties:
  - `filters.mimeTypes`: optional - list of mimetypes in the format specified in https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept#unique_file_type_specifiers
  - `filters.extensions`: optional - list of extensions, all extensions need to start with a dot and have the format `.ext`.
  - `filters.multiple`: false by default, whether to allow multiple files to be selected

The method returns a `Promise` that resolves to an Array of `File`-Objects.

Example:
```js
// then/catch
window.webxdc.importFiles({
  mimeTypes: ["text/calendar"],

>   extentions: [".ics"],

Reviewer 2: ```suggestion
  extensions: [".ics"],
```

}).then((files) => {
  /* do sth with the files */
}).catch((error) => {
    console.log(error);
});
// async/await
try {
  let files = await window.webxdc.importFiles({
    mimeTypes: ["text/calendar"],

>     extentions: [".ics"],

Reviewer 2: ```suggestion
    extensions: [".ics"],
```
